### PR TITLE
add metadata downloads in a DropDown

### PR DESCRIPTION
**Pull request by caravinci**
does this:
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "test": "npm run build && jest"
   },
   "dependencies": {
-    "@arranger/components": "^2.16.1",
+    "@arranger/components": "^2.17.0",
```
Comment (caravinci) [inline package.json:13]: first, update Arranger Components to the latest legacy version

File: components/pages/explorer/RepoTable/index.tsx [hunk 2]
```diff
@@ -379,36 +421,34 @@ const RepoTable = (props: PageContentProps): ReactElement => {
...
+  const customExporters = [
+    {
+      columns: tsvExportColumns,
+      fileName: `virusseq-metadata-export-${today}.tsv`,
+      function: 'saveTSV',
+      label: 'Metadata only',
+    },
+    { function: handleBundleDownload, label: 'Metadata & Fasta files' },
+  ];
+
```
Comment (caravinci) [inline components/pages/explorer/RepoTable/index.tsx:435]: use Arranger's own exporter customisation capabilities.

File: components/pages/explorer/RepoTable/index.tsx [hunk 3]
```diff
@@ -379,36 +421,34 @@ const RepoTable = (props: PageContentProps): ReactElement => {
     archviePromise.then(setArchive);
   };
 
-  const DownloadButton = (
-    <Button
-      onClick={handleDownload}
```
Comment (caravinci) [inline components/pages/explorer/RepoTable/index.tsx:384]: get rid of the old one

File: components/pages/explorer/RepoTable/index.tsx [hunk 4]
```diff
@@ -62,7 +60,10 @@ const getTableStyle = (theme: typeof defaultTheme) => css`
     }
 
     & .group {
+      display: flex;
```
Comment (caravinci) [inline components/pages/explorer/RepoTable/index.tsx:63]: and add a whole bunch of styling to make them look the same

File: components/pages/explorer/RepoTable/index.tsx [hunk 5]
```diff
@@ -342,15 +384,15 @@ const RepoTable = (props: PageContentProps): ReactElement => {
...
+      action: 'Archive Build',
     });
     return await startArchiveBuildBySetId(setId);
   };
 
   const updateArchiveState = async () => {
     if (archive?.status === 'BUILDING') {
       await sleep(5000);
-      await findArchvieById(archive.id).then(setArchive);
+      await findArchiveById(archive.id).then(setArchive);
```
Comment (caravinci) [inline components/pages/explorer/RepoTable/index.tsx:395]: All remaining changes are just improving this typo someone made in many files across the app

**Review by caravinci — COMMENTED** [review]
Some comments to assist reviewing this PR 👍

**Review by joneubank — APPROVED** [review]
RIP ARCHVIES

**Pull request merged**